### PR TITLE
Minor repository tweaks

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
@bparees @jhadvig fyi

I think the naming convention for the base image should be: `openshift/base-OSVERSION`

**Comment by bparees** [comment]
works for me, but we need to fix all the images referencing this, right?

**Comment by mfojtik** [comment]
there are none atm.

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -1,5 +1,11 @@
 FROM rhel7
 
+# This image is an base image for OpenShift v3 Docker images.
```
Comment (soltysh) [inline Dockerfile.rhel7]: Suggestion: This is base image for OpenShift...

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,12 @@
 FROM centos:centos7
 
+# This image is an base image for OpenShift v3 Docker images.
```
Comment (soltysh) [inline Dockerfile]: Suggestion: This is base image for OpenShift...

File: Dockerfile [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+# It includes all dependencies that users might need to debug broken
+# applications or services as well as common build dependencies for all
+# languages and frameworks.
+
+MAINTAINER Jakub Hadvig <jhadvig@redhat.com>
```
Comment (soltysh) [inline Dockerfile]: The best for this would be to have a devexp alias for that purpose, I guess.

File: Dockerfile (hunk 2 above)
Comment (bparees) [inline Dockerfile]: This is *a* base image for...... or *the* base image for.

File: Dockerfile.rhel7 (hunk 1 above)
Comment (bparees) [inline Dockerfile.rhel7]: this is a/the base image for

File: Dockerfile (hunk 2 above)
Comment (soltysh) [inline Dockerfile]: Yeah, the articles a, an and the are always a problem for me :wink:

File: Dockerfile (hunk 3 above)
Comment (jhadvig) [inline Dockerfile]: @soltysh this looks like the right approach but I doubt we have something like that.
@mfojtik thoughts?

**Pull request closed without merging**